pettingzoo/classic/rlcard_envs/texas_holdem.py

```diff
@@ -69,6 +72,7 @@
 
 ### Version History
 
+* v5: Added multiround option (1.x)
```

Comment:
this should be 1.24.2 as I don't want to do another major release for a while since there aren't any really breaking changes

Reply:
Ok